packages/dart/lib/src/network/parse_query.dart

```diff
@@ -216,20 +216,20 @@ class QueryBuilder<T extends ParseObject> {
       {bool caseSensitive = false}) {
     if (caseSensitive) {
       queries.add(MapEntry<String, dynamic>(
-          _SINGLE_QUERY, '"$column":{"\$regex": "$value"}'));
+          _singleQuery, '"$column":{"\$regex": "$value"}'));
     } else {
       queries.add(MapEntry<String, dynamic>(
-          _SINGLE_QUERY, '"$column":{"\$regex": "$value", "\$options": "i"}'));
+          _singleQuery, '"$column":{"\$regex": "$value", "\$options": "i"}'));
     }
   }
 
-  /// Powerful search for containing whole words. This search is much quicker than regex and can search for whole words including wether they are case sensitive or not.
+  /// Powerful search for containing whole words. This search is much quicker than regex and can search for whole words including weather they are case sensitive or not.
```

Comment:
```suggestion
  /// Powerful search for containing whole words. This search is much quicker than regex and can search for whole words including whether they are case sensitive or not.
```

Reply:
See change suggestion

Reply:
Sorry, I had not seen it
Thanks for the change